src/pages/instances/actions/snapshots/InstanceSnapshotActions.tsx

```diff
@@ -95,6 +96,13 @@ const InstanceSnapshotActions: FC<Props> = ({
           "u-snapshot-actions": !isDeleting && !isRestoring,
         })}
         items={[
+          <DuplicateInstanceSnapshotBtn
+            key="duplicate"
+            instance={instance}
+            snapshot={snapshot}
+            isDeleting={isDeleting}
+            isRestoring={isRestoring}
+          />,
```

Comment:
I sugges to change the order:
1. edit
2. restore
3. create image
4. create instance (this one)
5. delete snapshot

Reply:
I've adjusted the ordering.